defaults/config.js

```diff
@@ -368,4 +368,11 @@ module.exports = {
 	// @default  false
 	//
 	debug: false,
+
+	// Enables logging raw IRC messages into each server window.
+	//
+	// @type     boolean
+	// @default  false
+	//
+	debugRaw: false,
```

Comment:
Object is more explicit because you disable options without removing them. As with the list, you will need to find the options elsewhere (e.g. comment above).